frontend/pages/shared-components/text-input/text-input.module.css

```diff
@@ -0,0 +1,34 @@
+.inputContainer {
+  display: flex;
```

Comment:
lets try to use css-grid instead of flex, far more clean and declarative.

Reply:
Flexbox & Grid serve somewhat different purposes! I've always heard the advice when you're dealing more with simple, single-dimensional layouts (like I feel like we are here) that flexbox is preferable to Grid
And especially in regards to mobile responsiveness, I find flexbox to be more dynamic? But happy to discuss further!

Here's an article I found helpful!
https://css-tricks.com/css-grid-replace-flexbox/